libs/blocks/ost/ost.css

```diff
@@ -31,3 +31,7 @@ body.tool-ost .ost>div {
 .price-unit-type::before {
   content: " ";
 }
+
+.spectrum-TabsPanel-tabpanel_3e140b > div > div>:last-child{
```

Comment:
This selector seems way too specific - are we guarenteed that `spectrum-TabsPanel-tabpanel_3e140b` will always exist?

Looks like you could instead target the button directly:
`spectrum-Button_e2d99e` or `data-testid="productsTab/nextButton"`

Reply:
I have updated the code.